java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -179,4 +183,9 @@ public void setCurrentFile(File currentFile) {
     this.currentFile = currentFile;
   }
 
+  public void endOfAnalysis() {
+    if(!classesNotFound.isEmpty()) {
+      LOG.warn("Classes not found during the analysis : {}", classesNotFound.stream().collect(Collectors.joining(", ")));
```

Comment:
Maybe add brackets around the list of classes? 
```
"Classes not found during the analysis : [x.y.z.myClass, x.z.MyOtherClass, ...]"
```

I also wonder about the possible volume of data we may end up displaying (analysis poorly configured?). Maybe displaying only the first 50 classes, and if more display all of them in debug? Splitting hairs...